pkg/splunk/enterprise/finalizers_test.go

```diff
@@ -126,6 +126,23 @@ func splunkDeletionTester(t *testing.T, cr splcommon.MetaObject, delete func(spl
 					{MetaName: "*v1.PersistentVolumeClaim-test-splunk-pvc-stack1-var"},
 				}
 			}
+
+			switch cr.GetObjectKind().GroupVersionKind().Kind {
+			case "Standalone":
+				mockCalls["Get"] = append(mockCalls["Get"], spltest.MockFuncCall{MetaName: "*v3.Standalone-test-stack1"})
+				mockCalls["Get"] = append(mockCalls["Get"], spltest.MockFuncCall{MetaName: "*v3.Standalone-test-stack1"})
```

Comment:
Just for the sake brevity and uniformity, I think another approach(probably better) can be like this - 
```
funcCalls := []spltest.MockFuncCall{
    spltest.MockFuncCall{MetaName: "*v3.Standalone-test-stack1"},
    spltest.MockFuncCall{MetaName: "*v3.Standalone-test-stack1"},
}
mockCalls["Get"] = append(mockCalls["Get"], funcCalls...)
```
What do you think? :)